enterprise-metrics/CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Entries should include a reference to the Pull Request that introduced the chang
   - ruler_storage.backend used to be s3 but is now filesystem
   - alertmanager_storage.backend used to be s3 but is now filesystem
   - See [GEM 2.0 release notes](https://grafana.com/docs/enterprise-metrics/latest/release-notes/v2-0/) for more information
+- [FEATURE] Upgrade to [Grafana Enterprise Metrics v2.0.1](https://grafana.com/docs/metrics-enterprise/latest/downloads/#v200----april-13th-2022). #767
```

Comment:
Just need to update this link to point to the correct header for v2.0.1 once it exists

Reply:
I thought these were the release notes... mb

Reply:
will merge only after deploying the website